ci-operator/step-registry/baremetalds/e2e/baremetalds-e2e-workflow.yaml

```diff
@@ -5,6 +5,7 @@ workflow:
     - ref: baremetalds-packet-setup
     - ref: ipi-install-rbac
     - ref: baremetalds-devscripts-setup
+    - ref: baremetalds-e2e-conf
```

Comment:
I think it's safe to move this step into the `test` section

Reply:
Done